e2e-tests/insurance/cypress/e2e/journeys/your-buyer/outstanding-or-overdue-payments/outstanding-or-overdue-payments.spec.js

```diff
@@ -0,0 +1,131 @@
+import { headingCaption, field } from '../../../../../../pages/shared';
+import { PAGES } from '../../../../../../content-strings';
+import { YOUR_BUYER_FIELDS as FIELDS } from '../../../../../../content-strings/fields/insurance/your-buyer';
+import { INSURANCE_ROUTES } from '../../../../../../constants/routes/insurance';
+import { YOUR_BUYER as FIELD_IDS } from '../../../../../../constants/field-ids/insurance/your-buyer';
+import application from '../../../../../../fixtures/application';
+
+const CONTENT_STRINGS = PAGES.INSURANCE.YOUR_BUYER.OUTSTANDING_OR_OVERDUE_PAYMENTS;
+
+const {
+  ROOT,
+  YOUR_BUYER: { OUTSTANDING_OR_OVERDUE_PAYMENTS, FAILED_TO_PAY, CURRENCY_OF_LATE_PAYMENTS, TRADING_HISTORY },
+} = INSURANCE_ROUTES;
+
+const { TOTAL_AMOUNT_OVERDUE, TOTAL_OUTSTANDING_PAYMENTS, OUTSTANDING_PAYMENTS } = FIELD_IDS;
+
+const baseUrl = Cypress.config('baseUrl');
+
+const { BUYER } = application;
+
+context(
+  'Insurance - Your buyer - Outstanding or overdue payments - As an Underwriter, I want to know about any outstanding and/or overdue payments the buyer owes the exporter, So that I can better understand the level of risk when assessing the application',
+  () => {
+    let referenceNumber;
+    let url;
+    let failedToPayUrl;
+    let tradingHistoryUrl;
+
+    before(() => {
+      cy.completeSignInAndGoToApplication({}).then(({ referenceNumber: refNumber }) => {
+        referenceNumber = refNumber;
+
+        url = `${baseUrl}${ROOT}/${referenceNumber}${OUTSTANDING_OR_OVERDUE_PAYMENTS}`;
+        failedToPayUrl = `${baseUrl}${ROOT}/${referenceNumber}${FAILED_TO_PAY}`;
+        tradingHistoryUrl = `${baseUrl}${ROOT}/${referenceNumber}${TRADING_HISTORY}`;
+
+        cy.completeAndSubmitYourBuyerForms({ formToStopAt: 'currencyOfLatePayments', outstandingPayments: true, exporterHasTradedWithBuyer: true });
+
+        cy.assertUrl(url);
+      });
+    });
+
+    beforeEach(() => {
+      cy.saveSession();
+    });
+
+    after(() => {
+      cy.deleteApplication(referenceNumber);
+    });
+
+    it('renders core page elements', () => {
+      cy.corePageChecks({
+        pageTitle: CONTENT_STRINGS.PAGE_TITLE,
+        currentHref: `${ROOT}/${referenceNumber}${OUTSTANDING_OR_OVERDUE_PAYMENTS}`,
+        backLink: `${ROOT}/${referenceNumber}${CURRENCY_OF_LATE_PAYMENTS}`,
+      });
+    });
+
+    describe('page tests', () => {
```

Comment:
Need to add "renders a `save and back` button" tests here and for the "failed-to-pay" test. Not a blocker, i'll add these in my PR